pkg/workload/histogram/exporter/util.go

```diff
@@ -0,0 +1,69 @@
+// Copyright 2024 The Cockroach Authors.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+package exporter
+
+import (
+	"regexp"
+	"time"
+
+	"github.com/codahale/hdrhistogram"
+	"github.com/gogo/protobuf/proto"
+	prom "github.com/prometheus/client_model/go"
+)
+
+var (
+	invalidCharRegex      = regexp.MustCompile(`[^a-zA-Z0-9_]`)
+	invalidFirstCharRegex = regexp.MustCompile(`^[^a-zA-Z_]`)
+	summaryQuantiles      = []float64{50, 95, 99, 100}
+)
+
+func sanitizeOpenMetricsLabels(input string) string {
+	sanitized := invalidCharRegex.ReplaceAllString(input, "_")
+	sanitized = invalidFirstCharRegex.ReplaceAllString(sanitized, "_")
+	return sanitized
```

Comment:
We should probably throw error if entered metrics labels don't adhere to the open metrics guidelines, as opposed to updating the label based on the regex.

Reply:
IMO it's better we sanitise them rather than failing